pandas/io/formats/csvs.py

```diff
@@ -30,7 +30,7 @@ class CSVFormatter(object):
     def __init__(self, obj, path_or_buf=None, sep=",", na_rep='',
                  float_format=None, cols=None, header=True, index=True,
                  index_label=None, mode='w', nanRep=None, encoding=None,
-                 compression='infer', quoting=None, line_terminator='\n',
+                 compression='infer', quoting=None, line_terminator=None,
```

Comment:
this is going to need a doc-string versionchanged, and an update

Reply:
On closer look, #24290 covered this. In fact, the same PR may have corrected the entire issue as long as the `CSVFormatter` class is always called with `line_terminator=None`. 

I made a minor docs update to reflect that PR, let me know if you had something else in mind.

Reply:
I think the reference issue is off? in any event, your change looks fine.